### PR TITLE
fix(oatmeal): delete before writing to file

### PR DESCRIPTION
**Pull request by Trip-kun**
Seems like a reasonable fix? 

Unless you guys have any other ideas

## Summary by CodeRabbit

- **Bug Fixes**
	- Improved file handling by ensuring existing files are cleared before new data is written, preventing data overlap issues.

### CODE REVIEW

File: src/main/java/wtf/triplapeeck/oatmeal/FileRW.java [hunk 1]
```diff
@@ -20,6 +20,7 @@ public String readAll() throws IOException {
         return new String(bytes, StandardCharsets.UTF_8);
     }
     public void writeAll(String string) throws IOException {
+        Files.deleteIfExists(path);
         outputstream=Files.newOutputStream(path);
         outputstream.write(string.getBytes(StandardCharsets.UTF_8));
         outputstream.flush();
```
[the remark this answers is not in the document]
Comment (Baconing) [reply src/main/java/wtf/triplapeeck/oatmeal/FileRW.java:26]: "There can't be any bugs if there isn't any code"

[the remark this answers is not in the document]
Comment (silvanathecat) [reply src/main/java/wtf/triplapeeck/oatmeal/FileRW.java:26]: wow its just bitching at you

[the remark this answers is not in the document]
Comment (Trip-kun) [reply src/main/java/wtf/triplapeeck/oatmeal/FileRW.java:26]: It's actually a good point but it's more stuff to handle in the Error/Exception handling PR

**Review by silvanathecat — COMMENTED** [review]
0/10 stars not enough tires in my meal and the waiter tried to murder my brother

**Review by Baconing — APPROVED** [review]
As a hotfix this will probably work, however I do feel that there is a better way, but we should get this out so we have a working file writer.

**Comment by silvanathecat** [comment]
stop having gay sex in general before i delete your bunyn

**Pull request merged**